src/app/RouteConfig.ts

```diff
@@ -0,0 +1,66 @@
+interface PreviousActivityConfig {
+  prevStage: number;
+  prevActivity: number;
+}
+
+interface Routes {
+  [componentName: string]: PreviousActivityConfig
+}
+
+export const ActivityRoutes: Routes = {
+  HighlightActivityComponent: {
+    prevStage: 1,
+    prevActivity: 0
+  },
+
+  PcpolicyComponent: {
+    prevStage: 1,
+    prevActivity: 1
+  },
+
+  PicturePuzzleComponent: {
+    prevStage: 1,
+    prevActivity: 2
+  },
+
+  AnimatedVideoComponent: {
+    prevStage: 1,
+    prevActivity: 3
+  },
+
+  MalariaLifeCycleComponent: {
+    prevStage: 2,
```

Comment:
Isn't the previous stage supposed to be one less than the stage this activity is on?
So in this case shouldn't it be 1 instead of 2?

Reply:
Hmm lets revert this change. The menu page has only 3 bars for 3 activities in the stage.
This appears to have activity number as 4

Reply:
@Raxerz reverted it. Please see.